### ISSUE TITLE
Start up sounds overlap when turning on VISUAL ALERTS

### ISSUE THREAD

**Issue by atlantajoe21** [linuxmint/cinnamon#12124]
### Distribution

LMDE6

### Package version

6.0.4

### Graphics hardware in use

Advanced Micro Devices, Inc. [AMD/ATI] Lucienne

### Frequency

Always

### Bug description

When enabling VISUAL ALERTS by going to System Settings > Keyboard > Enable Visual Alerts, the "bing-a-bing" old style start up sound, and the new "Wong" start up sound play at the same time when booting the OS. Producing a "Wong" "Bing-a-bing" sound when starting up Linux Mint Debian Edition 6 Faye.

### Steps to reproduce

I turned off VISUAL ALERTS and RESTARTED my computer and the "Wong" start up sound plays as expected. But when I turn VISUAL ALERTS back on and RESTART my computer, I get both the "Bing-a-bing" and the "Wong" start up sounds playing at the same time. This oddity occurs in LMDE6.

### Expected behavior

I would suspect that I should only be hearing one startup sound. Either the "Bing-a-bing" or the "Wong" but not both.

### Additional information

I found this issue when enabling visual alerts in System Settings > Keyboard > Enable Visual Alerts. Turning this on plays the "Bing-a-bing" startup sound. When you restart the system with Enable Visual Alerts on, BOTH the "Bing-a-bing" and the "Wong" startup sounds play at the same time. I hope this describes this issues. Please, if you have any questions regarding this, feel free to contact me on GitHub.

**Comment by clefebvre** [comment]
Can you show me a screenshot of the setting for the visual alert?

**Comment by atlantajoe21** [comment]
Absolutely. And I think I made a mistake in directing you to the problem. I apologize. I think this screenshot should lead you right.

![Screenshot from 2024-03-20 09-09-07](https://github.com/linuxmint/cinnamon/assets/156608874/52d63bdc-30f3-450f-9719-a44f5721b275)

**Comment by atlantajoe21** [comment]
The start up sound happens when you turn on "Enable Visual Alerts." I hear the "bing-a-bing" start up sound. Then, when I restart my computer, I hear both startup sounds play at the same time.

**Comment by clefebvre** [comment]
How do you launch these settings?

**Comment by atlantajoe21** [comment]
From System Settings > Accessibility > Keyboard.

**Comment by clefebvre** [comment]
Ok that makes sense. Thanks.
